### DISCUSSION TITLE
openpyxl component import problems

### DISCUSSION THREAD

**Discussion by maevans12** [labs4capella/python4capella#83] [Q&A]
Has there been any issues with the use of openpyxl?  I'm running Capella 5.1.0, python interpreter 3.7 and when I try to run the sample script "import_physical_components_from_xlsx.py", or anything else that runs the "from openpyxl import *" statement, something doesn't return from completing this import statement.  In the Progress terminal, it seems that something in Sirius_API.py is hanging.  

![image](https://user-images.githubusercontent.com/18238582/139747509-51f8d456-6b35-4318-a26b-bb4123150ba2.png)

![image](https://user-images.githubusercontent.com/18238582/139747593-2ce29537-811e-4df9-b4c4-d10452f3a97d.png)

NOTE:  Yes the images show a different script name than "import_physical_components_from_xlsx.py".  I copied the contents of ""import_physical_components_from_xlsx.py" to a different script to debug in a different project.  Both scripts behave the same.

**Comment by maevans12** [comment]
BTW, I'm running EASE 0.8.0, python4capella 1.0.0

**Comment by maevans12** [comment]
I'm pretty sure that this issue is within the Capella (Eclipse) environment.  I wrote a quick script in a text editor that uses the import command and it runs fine.

**Reply by merlyn-true** [reply]
I think I am encountering the same issue as you. 
Would you be able to walk me through your workaround?
Thanks

**Comment by ylussaud** [comment]
There is a related [discussion](https://github.com/labs4capella/python4capella/issues/84).

Also you can try patching your environment as described in this [issue](https://github.com/labs4capella/python4capella/issues/112) about numpy.
